frontend/public/components/masthead-toolbar.jsx

```diff
@@ -22,6 +23,25 @@ const UpdatesAvailableButton = ({obj, onClick}) => {
     : null;
 };
 
+const onServiceMesh = (url, name, e) => {
+  e.preventDefault();
+  window.open(`${url}?namespace=${name}`, '_blank').opener = null;
+};
+
+const ServiceMeshItem = ({namespace}) => {
+  if (namespace.loaded) {
+    const ns = namespace.data;
+    const serviceMeshUrl = _.get(ns, 'metadata.annotations["openshift.servicemesh/url"]');
```

Comment:
We would want this to be something like

```
servicemesh.openshift.io/url
```

or maybe

```
kiali.openshift.io/url
```

Reply:
Ok, changed to 'servicemesh.openshift.io/url`.